yarn.lock

```diff
@@ -2689,6 +2689,17 @@
     sequelize "^6.28.0"
     typescript-collections "^1.3.3"
 
+"@helium/fanout-sdk@^0.1.2":
```

Comment:
Weird should we see an addition for the `@helium/voter-stake-registry-sdk` here?